packages/plugin-workspace-tools/sources/commands/focus.ts

```diff
@@ -90,8 +91,18 @@ export default class WorkspacesFocus extends BaseCommand {
         workspace.manifest.dependencies.clear();
```

Comment:
Do we need to clear to be evicted workspace manifests?

Reply:
Hi! I'm trying to add tests for this PR as I had a similar issue, I didn't understand your comment; do you mean that if we evict a workspace, it isn't needed to clear it's dependencies?

Reply:
Correct, if you remove the entire workspace you don't have to clear its dependencies first since the entire thing will be removed anyways